bin/pg_tapgen

```diff
@@ -176,22 +286,24 @@ sub has_table {
     '$schema', '$table',
     'Should have table $schema.$table'
 );\n\n";
+    $total_tests++;
     has_pk($schema, $table);
     columns_are($schema, $table);
 }
 
 sub has_pk {
     my ($schema, $table) = @_;
     my $fn = _hasc($schema, $table, 'p') ? 'has_pk' : 'hasnt_pk';
-    print "select $fn(
+    print "SELECT $fn(
     '$schema', '$table',
     'Table $schema.$table should have a primary key'
 );\n\n";
+    $total_tests++;
 }
 
 sub columns_are {
     my ($schema, $table) = @_;
-    print "SET search_path = '$schema';\n";
+    # print "SET search_path = '$schema';\n";
```

Comment:
I don't understand why you commented-out this line. This is the reason why the descriptions went away when you added the schema name as the explicit first argument to each function. Without the schema, the tests functions themselves generate reasonable default descriptions.